modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb

```diff
@@ -51,11 +49,15 @@ def bd_upload_body(auto_claim:, file_body:, doc_type:)
     def claim_bd_upload_document(claim, doc_type, pdf_path)
       ClaimsApi::BD.new.upload(claim:, doc_type:, pdf_path:)
     # Temporary errors (returning HTML, connection timeout), retry call
-    rescue Faraday::ParsingError, Faraday::TimeoutError => e
+    rescue Faraday::ParsingError, Faraday::TimeoutError, ::Common::Exceptions::BackendServiceException => e
       ClaimsApi::Logger.log('benefits_documents',
                             retry: true,
                             detail: "/upload failure for claimId #{claim&.id}: #{e.message}; error class: #{e.class}.")
-      raise e
+      if will_retry_status_code?(e)
+        raise e
+      else
+        {}
+      end
```

Comment:
this is probably for a different ticket but we still don't alert nor set any status' on this. If it were to not retry we would never know it "errored"? Again, probably for another ticket but worth bringing up. I think this is good 
 to be aware of for your current task as well @rockwellwindsor-va . 

I would say we should add a log line ([similar to this](https://github.com/department-of-veterans-affairs/vets-api/blob/f2e59267bbcdaef8a9bee006bc564c4910692476/modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb#L66)) for this particular PR and set some form of Slack notification or really anything to alert of a failure here in another ticket. Or queue off of a log search in DataDog every X minutes looking for the specific tag. Similar to what we're expecting for the dead queue. Let's define it to work across the board, so something similar to the tag we defined with retries exhausted so we can search specific for it. 

Retries exhausted used: `claims_api_retries_exhausted`
I would vote we use this a bit more dynamic and we align with something like: `claims_api_sidekiq_failure` and we can provide details of the specific Job and Claim ID so we can investigate.